custom_components/econet300/const.py

```diff
@@ -87,9 +87,20 @@
 #######################
 
 SENSOR_MAP_KEY = {
-    "ecoster": {
-        "ecoSterTemp1",
-        "ecoSterTemp2",
+    "ecoMAX360i": {
+        "PS",
+        "Circuit2thermostatTemp",
+        "TempClutch",
+        "Circuit3thermostatTemp",
+        "TempWthr",
+        "TempCircuit3",
+        "TempCircuit2",
+        "TempBuforUp",
+        "TempCWU",
+        "TempBuforDown",
+        "heatingUpperTemp",
+        "Circuit1thermostat",
+        "heating_work_state_pump4",
```

Comment:
why are there diferent namings ?

Reply:
For me also strange, because in endpoint different naming:
![image](https://github.com/user-attachments/assets/4188e9f7-8014-4b7e-98a2-cd23dc9ea125)